rabies/analysis_pkg/cpca/modeling.py

```
@@ -0,0 +1,203 @@
+        raise ValueError("Wt_n must be an integer or 'n'.") 
+    elif Wt_n<0:
+        raise ValueError("Wt_n must be minimum 0.") 
+        
+    n_prior = C_prior.shape[1]
+    sequence = ['s']*N_max
+    Cnet,Wnet,Cs,Ws,Ct,Wt,C,W = cpca_quick(X, C_prior, sequence, verbose=False)
+    
+    # generate the fitting report
+    n_optim_idx, fig_list = gen_report(X, C_prior, Cs, min_prior_sim=min_prior_sim, Dc_W_thresh=Dc_W_thresh, Dc_C_thresh=Dc_C_thresh)
+    
+    if n_optim_idx==None:
```

Flake8 (7.2.0)

[error] comparison to None should be 'if cond is None:'

(E711)